Resources/views/Core/add_block.html.twig

```diff
@@ -19,7 +19,7 @@
         {% if column_count > 1 %}style="width: {{ column_count*140 }}px;"{% endif %}
             >
         {% for group in groups|reverse %}
-            {% set display = (group.roles is empty or is_granted('ROLE_SONATA_ADMIN') ) %}
+            {% set display = (group.roles is empty or is_granted(role_sonata_admin()) ) %}
```

Comment:
Not sure how to solve this. Maybe introduce new a method `is_granted_admin` or something similar